### ISSUE TITLE
Support Kibana 6

### ISSUE THREAD

**Issue by CodingSpiderFox** [bitsensor/elastalert-kibana-plugin#26]
I suggest postponing support für Kibana 5 as Kibana 6 stable was already released yesterday

**Comment by RubieV** [comment]
Kibana 5 is now completely supported.

**Comment by RubieV** [comment]
Please check Kibana 6 support as of the recent PR https://github.com/bitsensor/elastalert-kibana-plugin/pull/23

**Comment by CodingSpiderFox** [comment]
great!

**Comment by CodingSpiderFox** [comment]
I will check whether it works with Kibana 6

**Comment by CodingSpiderFox** [comment]
Unfortunately I get an error message with the current revision of the kibana5 branch when installed into Kibana 6.0.0:

```
Fatal Error
Error: [$injector:unpr] Unknown provider: $$HashMapProvider <- $$HashMap <- $$animateQueue <- $animate <- $compile <- $$animateQueue http://errors.angularjs.org/1.6.5/$injector/unpr?p0=%24%24HashMapProvider%20%3C-%20%24%24HashMap%20%3C-%20%24%24animateQueue%20%3C-%20%24animate%20%3C-%20%24compile%20%3C-%20%24%24animateQueue (http://localhost:5601/bundles/commons.bundle.js?v=16070:35)

Version: 6.0.0
Build: 16070

Error: Error: [$injector:unpr] Unknown provider: $$HashMapProvider <- $$HashMap <- $$animateQueue <- $animate <- $compile <- $$animateQueue
http://errors.angularjs.org/1.6.5/$injector/unpr?p0=%24%24HashMapProvider%20%3C-%20%24%24HashMap%20%3C-%20%24%24animateQueue%20%3C-%20%24animate%20%3C-%20%24compile%20%3C-%20%24%24animateQueue (http://localhost:5601/bundles/commons.bundle.js?v=16070:35)
[2758]/window.onerror@http://localhost:5601/bundles/commons.bundle.js?v=16070:95:24726
```

**Comment by david-billings** [comment]
Hi, What is the status of this? Does the plugin support Kibana 6? The linked PR seems stale.

**Comment by darrenhaken** [comment]
Any update on this?

**Comment by panuinth** [comment]
Hopefully, it will support Kibana 6 soon

**Comment by drstokes** [comment]
I'm trying to install with kibana 6.1.3 - are there any plans for this anytime soon?

**Comment by davidkarlsen** [comment]
Ping

**Comment by RubieV** [comment]
Sorry, we will not be working on Kibana 6 support in the coming weeks / months. Feel free to start a PR.

**Comment by pabla** [comment]
Hi, I have fixed this plugin for Kibana 6.2.1, but it has no backward compatibility with 5.x.
https://github.com/pabla/elastalert-kibana-plugin/tree/kibana6.2.x

**Comment by martijnrondeel** [comment]
I've added experimental support for the latest 6.2 release, Kibana 6.2.3, in this [branch](https://github.com/bitsensor/elastalert-kibana-plugin/tree/6.2.3). Please report any issues.

**Comment by martijnrondeel** [comment]
For anyone interested, there's a rewritten plugin available that currently supports Kibana 6.3.1. More information is available here: https://github.com/bitsensor/elastalert-kibana-plugin/issues/56
